src/main/less/core.less

```diff
@@ -279,3 +279,15 @@ svg:not(:root) {
         }
     }
 }
+
+.no-data-text {
+    font-family: "Helvetica Neue";
```

Comment:
We should remove this so that the font-family is inherited. 
Consumers tend to bring in their own font based on their framework and code companion library.

Reply:
Removed